### PR TITLE
Adding new info-icon

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-4741

### CODE REVIEW

File: packages/icon-info/icon-info.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  render() {
+    return icon;
+  }
+}
+
+customElements.define('ia-icon-info', IAIconInfo);
+
+export default IAIconInfo;
```
Comment (iisa) [inline packages/icon-info/icon-info.js:29]:
```suggestion
export default IAIconInfo;

```

File: packages/icon-info/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { html } from 'lit-html';
+
+export default html`
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" aria-labelledby="infoTitleID infoDescID">
+  <title id="infoTitleID">Info icon</title>
+  <desc id="infoDescID">Informative icon</desc>
+  <!-- <g> -->
```
Comment (iisa) [inline packages/icon-info/index.js:7]:
```suggestion
```

File: packages/icon-info/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <path d="M480,253C478.3,129.3,376.7,30.4,253,32S30.4,135.3,32,259c1.7,123.7,103.3,222.6,227,221C382.7,478.3,481.7,376.7,480,253
+      z M256,111.9c17.7,0,32,14.3,32,32s-14.3,32-32,32c-17.7,0-32-14.3-32-32S238.3,111.9,256,111.9z M300,395h-88v-11h22V224h-22v-12
+      h66v172h22V395z" class="fill-color"/>
+  <!-- </g> -->
```
Comment (iisa) [inline packages/icon-info/index.js:11]:
```suggestion
```

**Review by iisa — APPROVED** [review]
looks good. just did some code cleanup there for ya.

**Comment by khareyash05** [comment]
@nsharma123 Could you pls let me know how to run `bin/bootstrap test.svg icon-name 'icon description'` . I am new to this repo and not able to understand. I have added the logo but not able to perform the next step

**Pull request merged**